keras_core/utils/dataset_utils.py

```diff
@@ -1,3 +1,11 @@
+import tensorflow as tf
```

Comment:
Another thing -- we should not import tf or torch at the top of the file since that would make them required dependencies. They should be imported when needed (e.g. only import torch when you need to process a torch dataset).

Reply:
umm ok but since they are being used in multiple function declaring there would allows us to have multiple import statements.

Reply:
Done for torch also importing `tf` from `module_utils`